core/db/default/spree/store_credit.rb

```diff
@@ -10,7 +10,11 @@
   environment: Rails.env,
 )
 
-Spree::StoreCreditType.create_with(priority: 1).find_or_create_by!(name: Spree.t("store_credit.expiring"))
-Spree::StoreCreditType.create_with(priority: 2).find_or_create_by!(name: Spree.t("store_credit.non_expiring"))
+Spree::StoreCreditType.create_with(priority: 1).find_or_create_by!(name: 'Expiring')
+Spree::StoreCreditType.create_with(priority: 2).find_or_create_by!(name: 'Non-expiring')
 
-Spree::ReimbursementType.create_with(name: Spree.t("store_credit.store_credit")).find_or_create_by!(type: 'Spree::ReimbursementType::StoreCredit')
+Spree::ReimbursementType.create_with(name: "Store Credit").find_or_create_by!(type: 'Spree::ReimbursementType::StoreCredit')
+
+Spree::StoreCreditCategory.find_or_create_by(name: 'Gift Card')
```

Comment:
@jhawthorn shouldn't we use the bang version of this method? I see it didn't in seeds.rb but the others in this file do use the bang versions.